bot/services/auth_service.py

```diff
@@ -0,0 +1,31 @@
+from config_reader import config
+
+from .api.api_request import make_post_request
+from .api.request_models import AuthCodePostRequest, AuthTokenPostRequest
+from .api.response_models import AuthTokenPostResponse
+
+
+async def post_auth_code_response(email: str) -> dict:
```

Comment:
```suggestion
async def post_auth_code_response(email: str) -> None:
```

Функция ничего не возвращает